ios/AllAboutOlaf.xcodeproj/project.pbxproj

```diff
@@ -753,6 +759,7 @@
 			children = (
 				742B362B86E443C09A7F9FD9 /* Entypo.ttf */,
 				E6FBE71A05CC43E2B4885180 /* Ionicons.ttf */,
+				3A8C0CF21F9EF65C0064B66C /* windmill.png */,
```

Comment:
Do we really only need one icon file?

Should/could the icon ingots into images.xcassets as an app icon, instead?

Reply:
I was trying to limit duplication of files. I would be fine creating a separate space for them. Also I am unsure how to reference the file if it’s in xcassets.

On another note, should we update the icon used in the credits view based off of the selected iOS icon?

Reply:
>On another note, should we update the icon used in the credits view based off of the selected iOS icon?

huh, we could do that, couldn't we. That'd be neat.

> Do we really only need one icon file?
I was mainly asking if we really only needed one file, because the main icon has to come in 24, 44, 96, 48, 1024, and some other sizes, for things like iPhone homescreen (@{1,2,3}x), iPad homescreen (@{1,2}x), Spotlight (@{1,2,3}x), Settings (@{1,2,3}x) and maybe others.